### PR TITLE
newnewnew

### PR DESCRIPTION
**Pull request by Andrew10x**

### CODE REVIEW

**Comment by FakeAccount17** [comment]
no

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var myObj = new Object(),
+    str = "myString",
+    rand = Math.random(),
+    obj = new Object();
+
+myObj.type              = "Dot syntax";
+myObj["date created"]   = "String with space";
+myObj[str]              = "String value";
+myObj[rand]             = "Random Number";
```
Comment (FakeAccount17) [inline js/script.js:9]: nononon

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var myObj = new Object(),
+    str = "myString",
+    rand = Math.random(),
```
Comment (FakeAccount17) [inline js/script.js:3]: well

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var myObj = new Object(),
+    str = "myString",
+    rand = Math.random(),
+    obj = new Object();
+
+myObj.type              = "Dot syntax";
+myObj["date created"]   = "String with space";
```
Comment (FakeAccount17) [inline js/script.js:7]: well

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    rand = Math.random(),
+    obj = new Object();
+
+myObj.type              = "Dot syntax";
+myObj["date created"]   = "String with space";
+myObj[str]              = "String value";
+myObj[rand]             = "Random Number";
+myObj[obj]              = "Object";
+myObj[""]               = "Even an empty string";
+
```
Comment (FakeAccount17) [inline js/script.js:12]: this is new comment

File: js/script.js (hunk 1 above)
Comment (Andrew10x) [reply js/script.js:9]: yes

**Comment by Andrew10x** [comment]
fk

File: js/script.js [hunk 5]
```diff
@@ -11,3 +11,19 @@ myObj[obj]              = "Object";
...
+
+
+var myObj = new Object(),
+    str = "mytring",
+    rand = Math.random(),
+    obj = new Object();
+
+myObj.type              = "Dot syntax";
+myObj["date created"]   = "String with space";
```
Comment (FakeAccount17) [inline js/script.js:23]: new dich

File: js/script.js [hunk 6]
```diff
@@ -11,3 +11,19 @@ myObj[obj]              = "Object";
...
+myObj[str]              = "String value";
+myObj[rand]             = "Random Number";
+myObj[obj]              = "Object";
+myObj[""]               = "Even an empty string";
```
Comment (FakeAccount17) [inline js/script.js:27]: very new

Comment (Andrew10x) [reply js/script.js:27]: fk

File: js/script.js [hunk 7]
```diff
@@ -11,3 +11,19 @@ myObj[obj]              = "Object";
...
+
+var myObj = new Object(),
+    str = "mytring",
+    rand = Math.random(),
+    obj = new Object();
+
+myObj.type              = "Dot syntax";
+myObj["date created"]   = "String with space";
+myObj[str]              = "String value";
+myObj[rand]             = "Random Number";
```
Comment (FakeAccount17) [inline js/script.js:25]: rrrrr

File: js/script.js [hunk 8]
```diff
@@ -11,3 +11,19 @@ myObj[obj]              = "Object";
...
+myObj[obj]              = "Object";
+myObj[""]               = "Even an empty string";
+
+console.log(myObj);
```
Comment (FakeAccount17) [inline js/script.js:29]: jjjjj

**Review by FakeAccount17 — CHANGES_REQUESTED** [review]

**Pull request merged**